instat/dlgBoxPlot.designer.vb

```diff
@@ -185,27 +185,27 @@ Partial Class dlgBoxplot
         Me.ucrPnlPlots.Size = New System.Drawing.Size(374, 35)
         Me.ucrPnlPlots.TabIndex = 0
         '
-        'ucrChkVarwidth
+        'ucrChkVarWidth
         '
-        Me.ucrChkVarwidth.Checked = False
-        Me.ucrChkVarwidth.Location = New System.Drawing.Point(10, 325)
-        Me.ucrChkVarwidth.Name = "ucrChkVarwidth"
-        Me.ucrChkVarwidth.Size = New System.Drawing.Size(135, 20)
-        Me.ucrChkVarwidth.TabIndex = 13
+        Me.ucrChkVarWidth.Checked = False
+        Me.ucrChkVarWidth.Location = New System.Drawing.Point(10, 299)
+        Me.ucrChkVarWidth.Name = "ucrChkVarWidth"
+        Me.ucrChkVarWidth.Size = New System.Drawing.Size(135, 20)
+        Me.ucrChkVarWidth.TabIndex = 12

```

Comment:
I switched chkVarWidth with chkHorizontal because VarWidth isn't always visible. This meant before when it was invisible and in the middle of checkboxes, there's a random huge gap between chks. I put the ucrChkVarWidth at the top so this gap isn't between chkboxes. This meant re-doing the tab order